templates/taxi/driver_list.html

```diff
@@ -6,8 +6,12 @@ <h1>Driver list</h1>
     <ul>
       {% for driver in driver_list%}
         <li>
-          <a href="{% url "taxi:driver_detail" pk=driver.id %}">{{ driver.username }}</a>
+            {% if user.pk == driver.pk %}
+                <a href="{% url "taxi:driver_detail" pk=driver.id %}">{{ driver.username }}</a> (Me)
+                {% else %}
+                <a href="{% url "taxi:driver_detail" pk=driver.id %}">{{ driver.username }}</a>
           ({{ driver.first_name }} {{ driver.last_name }})
+            {% endif %}
         </li>
       {% endfor %}
     </ul>
```

Comment:
Add a screenshot of this page.

Reply:
![image](https://user-images.githubusercontent.com/103201010/177334649-fcfc0eb8-7226-4c15-b80e-ffb48751e7d6.png)